components/org.wso2.carbon.identity.media.endpoint/pom.xml

```diff
@@ -0,0 +1,168 @@
+    <parent>
+        <groupId>org.wso2.carbon.identity.media</groupId>
+        <artifactId>identity-media</artifactId>
+        <version>2.0.4-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.carbon.identity.media.endpoint</artifactId>
+    <packaging>war</packaging>
+    <name>WSO2 Identity Server - Media Service Rest API</name>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>2.2</version>
+                <configuration>
+                    <webResources>
+                        <resource>
+                            <!-- this is relative to the pom.xml directory -->
+                            <directory>src/main/webapp</directory>
+                        </resource>
+                    </webResources>
+                    <warName>api#identity#media#v1.0</warName>
+                </configuration>
+            </plugin>
+<!--            <plugin>
+                <groupId>org.openapitools</groupId>
+                <artifactId>openapi-generator-maven-plugin</artifactId>
+                <version>4.1.2</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>${project.basedir}/src/main/resources/media_endpoint.yaml</inputSpec>
+                            <generatorName>org.wso2.carbon.codegen.CxfWso2Generator</generatorName>
+                            <configOptions>
+                                <sourceFolder>src/gen/java</sourceFolder>
+                                <apiPackage>org.wso2.carbon.identity.media.endpoint</apiPackage>
+                                <modelPackage>org.wso2.carbon.identity.media.endpoint</modelPackage>
+                                <packageName>org.wso2.carbon.identity.media.endpoint</packageName>
+                                <dateLibrary>java8</dateLibrary>
+                                <hideGenerationTimestamp>true</hideGenerationTimestamp>
+                            </configOptions>
+                            <output>.</output>
+                            <skipOverwrite>false</skipOverwrite>
+                        </configuration>
+                    </execution>
+                </executions>
+                <dependencies>
```

Comment:
remove unwanted dependency

Reply:
isn't this the recommended approach?
4th point given here: https://github.com/wso2/identity-api-server/blob/master/README.md